### ISSUE TITLE
plot_erp: layout legend_position not working

### ISSUE THREAD

**Issue by behinger** [unfoldtoolbox/UnfoldMakie.jl#225]
```
       plot_erp!(
       f[1,1], df,mapping = (;group=:duration,color=:duration),
       categorical_color=false,
       categorical_group=true,
       layout=(;legend_position=:bottom))
``` 
I wanted to move the legend to the bottom, but it is not working, legend stays at the right side

**Comment by behinger** [comment]
this is actually also true for `plot_erpimage!` - havent tested others

**Comment by vladdez** [comment]
use  legend = (; valign = :bottom,)

**Comment by behinger** [comment]
I think this is a remnant from the FaPra students. I think what they wanted to do is to put the position of the legend and it's orientation in one command. Right now you have to do:
`valign = :bottom, orientation = :horizontal`

that is, provide two commands to get a better legible legend - I think we should just provide an example where we specify both and remove the legend_position confusing argument. I dont think it is worth supporting (and obv. doesnt work anyway right now)

**Comment by vladdez** [comment]
fixed in #237
